### PR TITLE
Deprecated target api version

### PR DESCRIPTION
**Pull request by elyousfi5**

### CODE REVIEW

File: MOBILE_CLIENT/ANDROID/_LOW/DEPRECATED_TARGET_API_VERSION/meta.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "Target API level deprecation notice" : "https://support.google.com/googleplay/android-developer/answer/11926878?hl=en"
+  },
+  "title": "Deprecated Target API Version",
+  "cvss_v3_vector": "CVSS:3.0/AV:N/AC:L/PR:L/UI:R/S:C/C:L/I:L/A:N",
+  "privacy_issue": false,
+  "security_issue": true,
+  "categories": {
+    "PCI_STANDARDS":[
+      "REQ_6_2",
+      "REQ_6_3"
```
Comment (oussamaessaji) [inline MOBILE_CLIENT/ANDROID/_LOW/DEPRECATED_TARGET_API_VERSION/meta.json:15]: Are you sure that these requirements apply?

File: MOBILE_CLIENT/ANDROID/_LOW/DEPRECATED_TARGET_API_VERSION/meta.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "references": {
+    "Android API Levels": "https://developer.android.com/guide/topics/manifest/uses-sdk-element#min",
+    "Target API level deprecation notice" : "https://support.google.com/googleplay/android-developer/answer/11926878?hl=en"
+  },
+  "title": "Deprecated Target API Version",
+  "cvss_v3_vector": "CVSS:3.0/AV:N/AC:L/PR:L/UI:R/S:C/C:L/I:L/A:N",
+  "privacy_issue": false,
+  "security_issue": true,
+  "categories": {
+    "PCI_STANDARDS":[
```
Comment (3asm) [inline MOBILE_CLIENT/ANDROID/_LOW/DEPRECATED_TARGET_API_VERSION/meta.json:13]: That's it? No OWASP?

**Review by 3asm — CHANGES_REQUESTED** [review]

**Review by BlueSquare1 — COMMENTED** [review]
You can mention this in the description:

Starting on August 31, 2023:

New apps must target Android 13 (API level 33) or higher; except for [Wear OS](https://developer.android.com/wear) apps, which must target a version between Android 11 (API level 30) and Android 13 (API level 33), inclusive.

App updates must target Android 13 or higher and adjust for behavioral changes in Android 13; except for Wear OS apps, which must target Android 11.

Source: https://developer.android.com/google/play/requirements/target-sdk

File: MOBILE_CLIENT/ANDROID/_LOW/DEPRECATED_TARGET_API_VERSION/meta.json (hunk 1 above)
Comment (elyousfi5) [reply MOBILE_CLIENT/ANDROID/_LOW/DEPRECATED_TARGET_API_VERSION/meta.json:15]: Yeap

**Review by 3asm — APPROVED** [review]

**Pull request merged**